tests/output.rs

```diff
@@ -159,12 +159,34 @@ fn assert_failure_output(fixture_dir_name: &str, command_and_output: &str) {
         }));
 }
 
+fn assert_non_match_output(fixture_dir_name: &str, command_and_output: &str) {
+    let CommandAndOutput {
+        mut command_line_args,
+        output,
+    } = parse_command_and_output(command_and_output);
+    let command_name = command_line_args.remove(0);
+    Command::cargo_bin(command_name)
+        .unwrap()
+        .args(command_line_args)
+        .current_dir(get_fixture_dir_path_from_name(fixture_dir_name))
+        .assert()
+        .success()
+        .stdout(predicate::function(|stdout: &str| {
+            let stdout = massage_error_output(stdout);
+            stdout == output
+        }));
+}
+
 fn massage_error_output(output: &str) -> String {
     if cfg!(windows) {
         output.replace(".exe", "")
     } else {
         output.to_owned()
     }
+    .split('\n')
+    .map(|line| line.trim_end())
```

Comment:
There was some weird trailing whitespace being emitted by `clap` in the `--help` message (so strip it here)